workflows/misc_file_types/hobo_files.Rmd

```diff
@@ -0,0 +1,12 @@
+---
+title: "hobo-files"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+.hobo files are original files offloaded from a sensor, and show real-time graphs of temperature and soil moisture conditions.
```

Comment:
I think hobo files can probably contain more than just temp/soil moisture. Worth noting that these are raw data files.